samza-core/src/main/java/org/apache/samza/processor/SamzaContainerController.java

```diff
@@ -44,7 +44,7 @@
 public class SamzaContainerController {
   private static final Logger log = LoggerFactory.getLogger(SamzaContainerController.class);
 
-  private final ExecutorService executorService;
+  private ExecutorService executorService;
   private volatile SamzaContainer container;
```

Comment:
Good question. I would like to keep changes not associated with SEP-1 out of this PR.